sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/InsertRetryPolicy.java

```diff
@@ -33,7 +33,7 @@
    */
   public static class Context {
     // A list of all errors corresponding to an attempted insert of a single record.
-    TableDataInsertAllResponse.InsertErrors errors;
```

Comment:
BTW I think Java standards prefer adding a getter here over making the variable public.